js/main.js

```diff
@@ -31,7 +51,60 @@ const getRandomInclusive = (min, max, numberDigit) => {
   }
 
   return null;
+};
+
+// СОЗДАЕМ МАССИВ ОБЪЯВЛЕНИЙ
+const getRandomArrayElement = (items) => {
+  return items[getRandomIntInclusive(0, items.length - 1)];
 }
 
-getRandomIntInclusive(0, 10);
-getRandomInclusive(2, 10, 5);
+// перетасовка массива
+const shuffle = (items) => {
+
+  for (let i = items.length - 1; i > 0; i--) {
+    const j = getRandomIntInclusive(0, i);
+    [items[i], items[j]] = [items[j], items[i]];
+  }
+
+  return items;
+};
+
+// Массив случайной длинны с неповторяющимися элементами
+const getArrayRandomLenght = (items) => {
```

Comment:
Заодно поправь опечатку
```suggestion
const getArrayRandomLength = (items) => {
```